File: Modul_4/Soal_3/PRAK403-2310817210015-MuhammadBukhariFitri.py
input = input().split()

angka1, angka2 = map(int, input)

print(f"{angka1} {angka2}", end="")
if(angka1 < angka2) :
    for i in range(1, angka2 - angka1 + 1) :
        print(f" - {angka1 + i} {angka2 - i}", end="");

else :
    for i in range(1, angka1 - angka2 + 1) :
        print(f" - {angka1 - i} {angka2 + i}", end="")

File: Modul_4/Soal_3/PRAK403-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    int angka1, angka2;

    scanf("%d %d", &angka1, &angka2);
    printf("%d %d", angka1, angka2);

    if(angka1 < angka2) {
        for(int i = 1; i <= angka2 - angka1; i++) {
            printf(" - %d %d", angka1 + i, angka2 - i);
        }
    }
    else {
        for(int i = 1; i <= angka1 - angka2; i++) {
            printf(" - %d %d", angka1 - i, angka2 + i);
        }
    }
}